labelCloud/view/gui.py

```diff
@@ -187,6 +191,22 @@ def __init__(self, control: "Controller") -> None:
         self.timer.timeout.connect(self.controller.loop_gui)
         self.timer.start()
 
+    #open 2D(png) image
+    def open_2D_img(self):
+        tmp_cur_file = str(self.controller.pcd_manager.get_current_name().split(".")[0])
+        filename = r'C:/Users/user/Desktop/code_cloud/2d_button_cloud/pointclouds/' + tmp_cur_file + '.png'
```

Comment:
`image_path` would be more specific here. And to make it work on multiple platforms:
```suggestion
        image_path = os.path.join(
            config.get("FILE", "pointcloud_folder"),
            os.path.splitext(self.controller.pcd_manager.get_current_name())[0]
            + ".png",
        )
```

Reply:
Thank you for correcting me, I didn't consider any other platforms or OS.
It really helped me a lot 👍